src/app/components/practice/question-authoring-page/question-authoring-page.component.spec.ts

```diff
@@ -58,16 +67,28 @@ describe('QuestionAuthoringPageComponent', () => {
     fixture = TestBed.createComponent(QuestionAuthoringPageComponent);
     component = fixture.componentInstance;
     fixture.detectChanges();
-
-    // Setup all scenarios so that question title validation returns no errors
-    // unless otherwise specified
-    questionServiceStub.searchForQuestionBy.returns(of([]));
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  it('should create and not show the unauthorized error message if the user has permissions', () => {
+    const page = new QuestionAuthoringTestPage(fixture);
+    page.fixture.detectChanges();
+    expect(page.componentInstanceUnderTest).toBeTruthy();
+
+    expect(page.primaryFeatureArea).toBeTruthy('the primary feature area could not be seen');
+    expect(page.unauthorisedFeatureAccessMessage).toBeNull('the unauthorised feature access message could be seen');
   });
 
```

Comment:
The warning on assertion fail can cause confusion for the reader. These will be removed as part of #55 .